simplivity/resources/resource.py

```diff
@@ -246,7 +246,7 @@ def do_post(self, uri, data, timeout, custom_headers, flags=None):
 
         return self.task_affected_resources(task, timeout)
 
-    def do_put(self, uri, data, timeout, custom_headers):
+    def do_put(self, uri, data, timeout, custom_headers=None):
```

Comment:
These changes should be documented in the change.log file

Reply:
Updated the changelog file